cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/vsphere/VsphereManagementCluster.constants.tsx

```diff
@@ -6,7 +6,7 @@ export enum VSPHERE_FIELDS {
     DATACENTER = 'datacenter',
     IPFAMILY = 'ipFamily',
     USETHUMBPRINT = 'useThumbprint',
-    CLUSTERNAME = 'clusterName',
+    CLUSTERNAME = 'CLUSTER_NAME',
```

Comment:
nit: I'm thinking our new standard is to leave the value as `clusterName`; was there a particular reason you wanted `CLUSTER_NAME` instead?